components/inputs/input-date.js

```diff
@@ -61,6 +61,11 @@ class InputDate extends LabelledMixin(SkeletonMixin(FormElementMixin(LocalizeCor
 			 * Disables validation of max and min value. The min and max value will still be enforced but the component will not be put into an error state or show an error tooltip.
 			 */
 			noValidateMinMax: { attribute: 'novalidateminmax', type: Boolean },
+			/**
+			 * @ignore
+			 * Optionally add a 'Now' button to be used in date-time pickers only.
+			 */
+			nowButton: { attribute: 'now-button', type: Boolean },
```

Comment:
Added the ignore tag here since this option should be used for the date and time pickers together, and not for just date-pickers.

Reply:
Nit: I might suggest naming this something like `has-now` without exposing the fact that we're using a button to implement it. That way if we change the underlying implementation in the future, the attribute still makes sense. Although since it's internal it doesn't _really_ matter.